packages/x-date-pickers-pro/src/DateRangePicker/shared.tsx

```diff
@@ -41,7 +41,7 @@ export interface BaseDateRangePickerProps<TDate>
       BasePickerInputProps<DateRange<TDate>, TDate, 'day', DateRangeValidationError>,
       'view' | 'views' | 'openTo' | 'onViewChange' | 'orientation'
     >,
-    ExportedDateRangeCalendarProps<TDate>,
+    Omit<ExportedDateRangeCalendarProps<TDate>, 'calendars'>,
```

Comment:
We should be able to remove this Omit here as discussed last week (sorry if you just did not have the time to do it yet)

Reply:
Could you remind me about this discussion? 🙈 
If we remove this Omit, the Mobile Picker variants get the `calendars` prop.
Would you suggest moving this `Omit` to `Mobile` variant types? 🤔

Reply:
Forget what I said. 🙈 
Moved the `calendars` prop to the `DateCalendarProps` as discussed. 👌

Reply:
The holidays did not help here :laughing: 
Thanks for the fix :+1: